frontend/asfui/authentication/views.py

```diff
@@ -30,9 +30,8 @@ def login_view(request):
             else:    
                 msg = 'Invalid credentials'    
         else:
-            msg = 'Error validating the form'    
-
-    return render(request, "accounts/login.html", {"form": form, "msg" : msg})
+            msg = 'Error validating the form'      
```

Comment:
Please consider adding some error loggers here.

Reply:
error displayed not required.